src/main/java/com/locallampoon/fiveh/core/Player.java

```diff
@@ -216,10 +218,10 @@ void flee(Map<String, Room> houseMap) {
 
     void getCurrentItemDetails() {
```

Comment:
Nothing to do with you, but I just noticed that this never gets triggered. All inventories start as a 5-length array of "", so they are never empty.

Reply:
Ok. few functions in our code never got triggered. If it is created by me, i will delete it